apps/daimo-mobile/src/view/shared/Icons.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { ViewStyle } from "react-native";
+import { Path, Svg } from "react-native-svg";
+
+export const ClockIcon = ({ style }: { style?: ViewStyle }) => (
+  <Svg width="24" height="25" viewBox="0 0 24 25" fill="none" style={style}>
```

Comment:
minor: I think because the Figma was Hug x Hug rather than fixed width, the frame export made the height 25 instead of 24. In future, you can change it to fixed width in the right sidebar before exporting to make it even.

Reply:
cc @michelle-chiu